### DISCUSSION TITLE
en_core_web_sm load error:  Config validation error  tokenizer -> @tokenizers   a class is expected

### DISCUSSION THREAD

**Discussion by yuxy411** [explosion/spaCy#12076] [Help: Installation]
when I load the en_core_web_sm, I got a bug as Config validation error, tokenizer -> @tokenizers   a class is expected.
`import spacy
nlp = spacy.load('en_core_web_sm')`
Is this something wrong with my code?

* Operating System: linux
* Python Version Used: python 3.8
* spaCy Version Used: 3.1.0

**Comment by shadeMe** [comment]
Are you using the correct version of the `en_core_web_sm` model? Each release of spaCy comes with its own pre-trained models. 

You can execute the following command in a bash shell to get a report of potential incompatibilities:
```
python -m spacy validate
```

To install the correct version of the model, you can use the following command:
```
python -m spacy download en_core_web_sm
```
